keyboards/waldo/waldo.h

```diff
@@ -0,0 +1,51 @@
+#ifndef WALDO_H
+#define WALDO_H
+
+#include "quantum.h"
+
+#define KEYMAP_ANSI( \
```

Comment:
In addition to mechmerlin's comment, this also seems to be a standard layout 60% PCB. Line 6 should be changed to:

```suggestion
#define LAYOUT_60_ansi( \
```